impl/baseline/failing_tests_text_format.txt

```diff
@@ -405,3 +405,10 @@ Required.Proto3.TextFormatInput.Uint64FieldMaxValueOctal.ProtobufOutput
 Required.Proto3.TextFormatInput.Uint64FieldMaxValueOctal.TextFormatOutput
 Required.Proto3.TextFormatInput.Uint64FieldTooLargeHex
 Required.Proto3.TextFormatInput.Uint64FieldTooLargeOctal
+Required.Proto2.TextFormatInput.ClosedEnumFieldByNumber.ProtobufOutput # Failed to parse input or produce output.
```

Comment:
The comment at the end is added by the conformance tool, right? Nice small improvement.

Reply:
It is yeah. I agree.